lib/team.rb

```diff
@@ -32,17 +32,23 @@ def provide_info
   end
 
   def win_percentage(games)
-    number_of_games_won(games)/games_played_in(games).count.to_f * 100
+    (number_of_games_won(games)/games_played_in(games).count.to_f * 100).round(2)
   end
 
-  def number_of_games_won(games)
-    games_won = 0
-    games.each do |game|
-      if game.away_team_id == @team_id && game.outcome.include?("away") || game.home_team_id == @team_id && game.outcome.include?("home")
-        games_won += 1
-      end
+  def games_won(games)
```

Comment:
this looks better! I think once we jump into some more refactoring I could also use that for the home and away win percentages stuff